styles/styles.css

```
@@ -0,0 +1,658 @@
+    align-items: center;
+    text-align: center;
+    text-transform: uppercase;
+    color: #FFFFFF;
+}
+
+.social-link-vk {
+    background-image: url(../images/vk.png);
+    background-repeat: no-repeat;
+    background-size: contain;
+    width: 24px;
+}
+
+.social-link-tg {
+    background-image: url(../images/telegram.png);
+    background-repeat: no-repeat;
+    background-size: contain;
+    width: 18px;
+}
+
+.social-link-youtube {
+    background-image: url(../images/youtube.png);
+    background-repeat: no-repeat;
+    background-size: contain;
+    width: 22px;
+}
+.social-list {
+    display: flex;
+    justify-content: flex-start;
+}
+
+.button-official-site {
+    background-image: url(../images/html-academy.png);
+    background-repeat: no-repeat;
+    background-size: contain;
+    width: 115px;
+    height: 33px;
+}
+
+.breadcrumbs-link-main {
+    background-image: url(../images/house.svg);
+    width: 12px;
+    height: 12px;
+    background-repeat: no-repeat;
+    background-size: contain;
+}
+
+.button-take-it {
+    font-family: inherit;
+    text-transform: uppercase;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 20px;;
+    background: #82B3D3;
+    color: #FFFFFF;
+    text-decoration: none;
+    border-radius: 4px;
+    padding: 8px 50px;
+}
+
+.button-out {
+    font-family: inherit;
+    text-transform: uppercase;
+    font-weight: 700;
+    color:#FFFFFF;
+    background-color: rgba(130, 179, 211, 0);
+    font-style: normal;
+    font-size: 16px;
+    line-height: 20px;
+    align-items: center;
+    text-align: center;
+    border-radius: 4px;
+    padding: 8px 50px;
+}
+
+.catalog-head {
+    font-weight: 700;
+    font-size: 30px;
+    line-height: 36px;
+    text-transform: uppercase;
+    color: #000000;
+}
+
+.sorting {
+    font-family: inherit;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 18px;
+    line-height: 21px;
+    color: #333333;
+}
+
+.type-of-hotel,
+.price {
+    font-size: 18px;
+    line-height: 21px;
+    color: #333333;
+}
+
+
+
+.raiting {
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 20px;
+    text-align: center;
+    text-transform: uppercase;
+    color: #333333;
+    width: 140px;
+    min-height: 37px;
+    background: #F2F2F2;
+    border-radius: 4px;
+    padding-top: 9px;
+    padding-bottom: 8px;
+    padding-left: 23px;
+    padding-right: 22px;
+}
+
+.hotel-name {
+    font-family: inherit;
+    font-weight: 700;
+    font-size: 24px;
+    line-height: 28px;
+    color: #000000;
+}
+
+.pagination-link-current,
+.pagination-link-next {
+    width: 60px;
+    height: 60px;
+    background: #F2F2F2;
+    border-radius: 4px;
+    font-family: inherit;
+    color: #000000;
+}
+
+.pagination-link {
+    width: 60px;
+    height: 60px;
+    background: #82B3D3;;
+    border-radius: 4px;
+    color: #FFFFFF;
+}
+
+
+.page-footer{
+    background-color: #FFFFFF;
+    justify-content: space-evenly;
+}
```

```suggestion
.page-footer{
    background-color: #FFFFFF;
    justify-content: space-evenly;
}
```

`space-evenly` распределяет элементы вдоль главной оси, но справа и слева тоже будут расстояния. Причем эти расстояния будут зависеть от количества и размера элементов. 
Но по макету крайние отступы фиксированы и не должны меняться в зависимости от содержимого. 
И эти расстояния это паддинги самого футера. 

Так что `space-evenly` тут не подходит.